docs/pages/RPC/+Page.mdx

```diff
@@ -151,8 +150,8 @@ RPC enables your frontend to tap directly into the full power of the server such
 
 Let's see what happens when a telefunction is called.
 
-```js
-// hello.telefunc.js
+```ts
+// hello.telefunc.ts
```

Comment:
Let's add a type to `async function hello({ name }) {` below, and let's do it to every `.telefunc.ts` file.